tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -118,6 +118,25 @@ def test_concatenate_list(self):
             == "Milk, Eggs, Flour, Water"
         )
 
+    def test_add(self):
+        assert self.transforms.add(1, 2) == 3
+        assert self.transforms.add("1", 2) == 3
+
+    def test_format_ordinal_prefix(self):
```

Comment:
The test method should be named to match the behaviour, something like `test_format_ordinal_with_determiner`.

Reply:
renamed to "test_format_ordinal_with_determiner" and "test_format_ordinal_without_determiner". 👍